src/Windows/Avalonia.Win32/WindowImpl.AppWndProc.cs

```diff
@@ -1132,6 +1132,20 @@ private void UpdateInputMethod(IntPtr hkl)
             Imm32InputMethod.Current.SetLanguageAndWindow(this, Hwnd, hkl);
         }
 
+        /// <summary>
+        /// Get the location of the pointer in himetric units.
+        /// </summary>
+        /// <param name="info">The pointer info.</param>
+        /// <returns>The location of the pointer in himetric units.</returns>
+        private Point GetHimetricLocation(POINTER_INFO info)
+        {
+            GetPointerDeviceRects(info.sourceDevice, out var pointerDeviceRect, out var displayRect);
+            var himetricLocation = new Point(
+                info.ptHimetricLocationRawX * displayRect.Width / (double)pointerDeviceRect.Width,
+                info.ptHimetricLocationRawY * displayRect.Height / (double)pointerDeviceRect.Height);
```

Comment:
Shouldn't it include rect X and Y as well?
Example, pen tablet can be configured to have narrow active rect inside of its working area.

![image](https://github.com/user-attachments/assets/7e382666-e0c8-47a8-ba05-59946b4c2a67)

Reply:
I am not sure how `GetPointerDeviceRects` API handles that, so it might be included already.

Reply:
Thank you for your reminder. I will test the situation of multiple screens and then make appropriate modifications to the code.